testing/no-std-tests/src/main.rs

```diff
@@ -42,21 +42,27 @@ fn compile_test() {
     const METADATA: &[u8] = include_bytes!("../../../artifacts/polkadot_metadata_small.scale");
     subxt_metadata::Metadata::decode(&mut &METADATA[..]).expect("should be valid metadata");
 
-    // Subxt Signer compiles:
-    use subxt_signer::sr25519;
-    let keypair = sr25519::dev::alice();
-    let message = b"Hello!";
-    let _signature = keypair.sign(message);
-    let _public_key = keypair.public_key();
+    // Subxt signer compiles (though nothing much works on this particular nostd target...):
```

Comment:
maybe add a link to https://docs.rs/getrandom/latest/getrandom/#supported-targets here